<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1407</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление научными исследованиями по тематическому плану</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>1388</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1381</width>
        <height>651</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_add_widget">
     <widget class="QWidget" name="Tp_nir_add_widget" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-20</y>
        <width>1381</width>
        <height>651</height>
       </rect>
      </property>
      <widget class="QTextEdit" name="Tp_nir_add_grntiNumber_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Tp_nir_add_grntiNature_comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiHead_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiCode_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiName_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiHeadPost_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_plannedFinancing_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Tp_nir_add_widget_saveButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_widget_close_pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>390</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
      <widget class="QLabel" name="cod">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Код ВУЗа/название ВУЗа</string>
       </property>
      </widget>
      <widget class="QLabel" name="num_nir">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>70</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="character">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>110</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Характер НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="prepod_name">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>150</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Руководитель НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="grnti_cod">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>190</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Коды ГРНТИ</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_nir">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>230</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Название НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="prepod_job">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>270</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Должность руководителя НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="plan_fin">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>310</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Плановое финансирование НИР</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Tp_nir_add_VUZcode_name_comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_redact_widget">
     <widget class="QWidget" name="redact_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1381</width>
        <height>601</height>
       </rect>
      </property>
      <widget class="QPushButton" name="redact_widget_close_pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>400</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>310</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Должность руководителя НИР</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiNumber_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>230</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Коды ГРНТИ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>110</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер НИР</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiName_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_plannedFinancing_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="redact_widget_saveButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>190</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Руководитель НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>270</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Название НИР</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Tp_nir_redact_grntiNature_comboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiCode_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiHeadPost_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Код ВУЗа</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_add_grntiHead_textEdit_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>150</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Характер НИР</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>350</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Плановое финансирование НИР</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_redact_VUZshortName_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>70</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Название ВУЗа</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="Tp_nir_redact_VUZcode_textEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>1101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_complex_filter_widget">
     <widget class="QComboBox" name="Federal_District_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1021</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Federation_subject_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>1021</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="City_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>1021</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="VUZ_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>1021</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Федеральный округ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>60</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Субъект федераци</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Город</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ВУЗ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
     <widget class="QPushButton" name="close_complex_filter_pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Tp_nir_redact_widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>1391</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="add_widget_open_pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ДОБАВИТЬ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redact_widget_open_pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>РЕДАКТИРОВАТЬ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget_del_pushButton">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>УДАЛИТЬ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget_add_grnti_cod_pushbutton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ДОБАВИТЬ КОД ГРНТИ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget_filter_grnti_cod_pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ФИЛЬТРАЦИЯ ПО КОДУ ГРНТИ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="widget_hard_filter_pushButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>СЛОЖНЫЙ ФИЛЬТР</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>Tp_nir_redact_widget</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1407</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>Отображение таблиц</string>
    </property>
    <addaction name="action_show_Tp_nir"/>
    <addaction name="action_show_Tp_fv"/>
    <addaction name="separator"/>
    <addaction name="action_show_VUZ"/>
    <addaction name="action_show_grntirub"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Работа с бд</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Анализ</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Тип выделения значений</string>
    </property>
    <addaction name="Select_rows_action"/>
    <addaction name="Select_columns_action"/>
    <addaction name="Select_items_action"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_show_VUZ">
   <property name="text">
    <string>Информация по ВУЗам</string>
   </property>
  </action>
  <action name="action_show_Tp_nir">
   <property name="text">
    <string>информация по НИР</string>
   </property>
  </action>
  <action name="action_show_grntirub">
   <property name="text">
    <string>Рубрики НИР</string>
   </property>
  </action>
  <action name="action_check">
   <property name="text">
    <string>Проверка целостности</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>Добавление записи</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Редактирование</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Удаление запси</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Формирование документов</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Сохранение условий фильтрации</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Сформировать распоряжение</string>
   </property>
  </action>
  <action name="add_cod_grnti">
   <property name="text">
    <string>Редактирование кода грнти</string>
   </property>
  </action>
  <action name="filter_grnti">
   <property name="text">
    <string>Фильтрация по коду ГРНТИ</string>
   </property>
  </action>
  <action name="Select_rows_action">
   <property name="text">
    <string>По строкам</string>
   </property>
  </action>
  <action name="Select_columns_action">
   <property name="text">
    <string>По столбцам</string>
   </property>
  </action>
  <action name="Select_items_action">
   <property name="text">
    <string>Отдельная ячейка</string>
   </property>
  </action>
  <action name="action_show_Tp_fv">
   <property name="text">
    <string>Финансирование ВУЗов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
